### PR TITLE
wcslib: init at 4.25

### PR DESCRIPTION
**Pull request by bnikolic**

### CODE REVIEW

File: pkgs/development/libraries/wcslib/default.nix [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{ fetchurl,  stdenv, flex }:
+
+
+stdenv.mkDerivation rec {
+  name = "wcslib-4.25";
+
+  buildInputs = [ flex ];
+
+  src = fetchurl {
+    url = "ftp://ftp.eso.org/pub/dfs/pipelines/libraries/wcslib/wcslib-4.25.tar.bz2";
```
Comment (nckx) [inline pkgs/development/libraries/wcslib/default.nix]: If you're using `rec` above (currently for no reason), we might as wel use ${name} here.

File: pkgs/development/libraries/wcslib/default.nix [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{ fetchurl,  stdenv, flex }:
+
+
```
Comment (nckx) [inline pkgs/development/libraries/wcslib/default.nix]: Nitpick: no need for two blank lines.

File: pkgs/development/libraries/wcslib/default.nix [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  buildInputs = [ flex ];
+
+  src = fetchurl {
+    url = "ftp://ftp.eso.org/pub/dfs/pipelines/libraries/wcslib/wcslib-4.25.tar.bz2";
+    sha256 ="0iqwr318z01jp1pxj349bzmzp2mc6rpl0bnsn9nvsa97pnzf18g4"  ;
+  };
+
+  meta = {
+    description = ''World Coordinate System Library,
```
Comment (nckx) [inline pkgs/development/libraries/wcslib/default.nix]: This is quite long. I'd split it into `description`/`longDescription`.

File: pkgs/development/libraries/wcslib/default.nix [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    a standard library for coordinate system transformations in astronomy'';
+  };
```
Comment (nckx) [inline pkgs/development/libraries/wcslib/default.nix]: Would be nice to flesh this out a bit (licence, homepage, platforms...)

**Comment by bnikolic** [comment]
I've made the improvements. Did not yet add the home page as the official page seems down right now, will wait too see if it comes back.

**Comment by nckx** [comment]
Would that be http://www.atnf.csiro.au/people/mcalabre/WCS/wcslib/? That works fine here, but mentions 5.12 as current version so I'm not sure.

**Comment by nckx** [comment]
If it is the same library at an older version, please add a short explanation why in the commit.

**Comment by nckx** [comment]
@bnikolic: any news?

**Comment by bnikolic** [comment]
I've updated to most recent version (now that the official FTP site with the most recent version is reliably available again).

**Comment by nckx** [comment]
Could you squash all commits into one, rebasing onto current master (to shut up Travis)?

File: pkgs/development/libraries/wcslib/default.nix [hunk 5]
```diff
@@ -0,0 +1,25 @@
+{ fetchurl,  stdenv, flex }:
+
+stdenv.mkDerivation rec {
+  ver = "5.12";
```
Comment (nckx) [inline pkgs/development/libraries/wcslib/default.nix]: Just use `version` here, no need to abbreviate.

**Comment by bnikolic** [comment]
Re- baselined again, implemented the suggested change

**Comment by nckx** [comment]
Excellent, thanks.

**Pull request merged**